Assets/Scripts/Scenes/InGame/Stage/StageGenerator.cs

```diff
@@ -21,6 +21,7 @@ public class StageGenerator : MonoBehaviour
     {
         [SerializeField, Tooltip("int�ŃX�e�[�W�̏c���̃T�C�Y")]
         private Vector2 _stageSize;
+        public static Vector3 SpawnPosition;
```

Comment:
```suggestion
private Vector3 _spawnPosition;
public Vector3 spawnPosition { get => _spawnPosition;}
``` 
基本的にprivateにしてください。
静的にする必要がない場合はstaticを使わないでください。バグの原因になります。
privateを他スクリプトから参照する際はカプセル化を用いて、間接的にgetしてください。
カプセル化の例.
```
public Vector3 spawnPosition { get => _spawnPosition;}
```